### DISCUSSION TITLE
How to joint inversion Dispersion Curve-HVSR ?

### DISCUSSION THREAD

**Discussion by xdxal** [gimli-org/pyGIMLi#745] [General]
How to joint inversion Dispersion Curve-HVSR ?

**Comment by makeabhishek** [comment]
Dear @xdxal, Your query has been answered #744. it is always good to share a detailed description so that a broader audience can understand and contribute..

**Comment by geosaipod** [comment]
using following program : https://w3.ual.es/GruposInv/hv-inv/

**Comment by halbmy** [comment]
Unfortunately, this is not really open-source if you have to fill in a form or email to the authors to get the code.
